test/test_ops.py

```diff
@@ -339,6 +339,91 @@ def test_out(self, device, dtype, op):
         op(*sample.input, *sample.args, **out_kwargs)
         self.assertEqual(expected, out)
 
+    @ops([op for op in op_db if op.aliases])
+    def test_aliases_consistency_eager(self, device, dtype, op):
```

Comment:
Currently, there is a code duplication based on `test_variant_consistency_eager` to check forward only results.